descriptors/ModuleDescriptor-template.json

```diff
@@ -296,7 +296,7 @@
     },
     {
       "id": "scheduled-notice-storage",
-      "version": "0.1",
+      "version": "0.3",
```

Comment:
Should it be "0.2" instead of "0.3"?

Reply:
Actually, this pull request contains changes from https://github.com/folio-org/mod-circulation-storage/pull/186, where the version was changed from `0.1` to `0.2`. And this PR introduces new changes in the schema.